### ISSUE TITLE
Are you sure it's free?

### ISSUE THREAD

**Issue by yuanjian19900610** [rany2/edge-tts#259]
This seems to encapsulate a network request interface. The actual synthesis is also done by Microsoft services. If it is used commercially, it should be limited.

**Comment by rany2** [comment]
Yes, it depends on the web service used by the Microsoft Edge browser for TTS. The web service is available free of charge in the Edge browser and this library makes use of that.
